app/src/main/java/org/simple/clinic/sync/indicator/SyncIndicatorViewController.kt

```diff
@@ -41,38 +41,57 @@ class SyncIndicatorViewController @Inject constructor(
 
   private fun updateIndicatorView(events: Observable<UiEvent>): Observable<UiChange> {
     val screenCreated = events.ofType<SyncIndicatorViewCreated>()
-    val syncStateStream = lastSyncState
+    val lastSyncedStateStream = lastSyncState
         .asObservable()
         .distinctUntilChanged()
-        .filter { it.lastSyncProgress != null }
 
-    return Observables
-        .combineLatest(screenCreated, syncStateStream)
+    val syncProgress = lastSyncedStateStream.filter { it.lastSyncProgress != null }
+
+    val defaultSyncIndicatorState = lastSyncedStateStream
```

Comment:
Can you move this chain above the `syncProgress` chain because the two aren't related.

Reply:
Done.